src/averageAges.js

```diff
@@ -20,6 +20,28 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  people.forEach(function(person) {
+    person.age = person.died - person.born;
+  });
+
+  let men = [];
+
+  people.forEach(function(person) {
+    if (person.sex === 'm') {
+      men.push(person);
+    };
+  });
+
+  if (arguments.length === 2) {
+    men = men.filter(person => (
+      Math.ceil(person.died / 100) === century));
+  }
```

Comment:
Let's use &&, || or ? instead of `if ()` statement

Reply:
I don't get how to change it. Because there I have only one action for condition true, I tried to change it and that's what I got

```
  (arguments.length === 2)
    ? men = men.filter(person => (
      Math.ceil(person.died / 100) === century))
    : **?????**;
```

Reply:
If that is confusing you can move conditions to variables like: 
```
const men = people.filter(person => {
	const isMan = person.sex === 'm';
	const diedInCentury = Math.ceil(person.died / 100) === century;
	return isMan && (!century || diedInCentury);
});
```